tests/expectations/test_expectation_atomic_renderers.py

```diff
@@ -722,7 +722,7 @@ def test_atomic_prescriptive_summary_expect_column_values_to_be_increasing(
         "expectation_type": "expect_column_values_to_be_increasing",
         "kwargs": {
             "column": "my_column",
-            "strictly": 10,
+            "strictly": True,
```

Comment:
This Expectation kwarg does not accept integers. See "Keyword Args" for this Expectation [here](https://greatexpectations.io/expectations/expect_column_values_to_be_increasing?filterType=Backend%20support&gotoPage=1&showFilters=false&viewType=Summary).